ansible/inventory.yml

```diff
@@ -326,8 +321,6 @@ hosts:
 
     - scaleway:
         ubuntu1804-armv7l-1: {ip: 212.47.233.202}
```

Comment:
`test-scaleway-ubuntu1804-armv7l-1` is still reachable and the Jenkins agent for it is online. I don't believe it's being used for anything though. Should we remove this too, based on https://github.com/nodejs/build/issues/2237#issuecomment-762576849?

Reply:
pull out all the scaleway hosts, we've stopped using them and I need to retire them, _or_ find a way to repurpose them - Scaleway have been a good sponsor and it'd be nice to figure out how we might include them somehow (probably just a TODO for me - there's potential to get greater access to their growing general infra rather than this narrow armv7 infra which they've been trying to retire anyway)

Reply:
Okay removed all of them.